src/lua/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+MESSAGE(STATUS "Searching for Lua library --\n")
+FIND_PACKAGE(Lua)
+IF(NOT LUA_FOUND)
+	MESSAGE(FATAL_ERROR
+		"You need the Lua development library to be able to compile with Lua support.\nSee INSTALL file for details and more info\n"
+	)
+ENDIF(NOT LUA_FOUND)
+
+ADD_DEFINITIONS(-DLUA)
+
+ADD_CUSTOM_TARGET(
+	lua_bindings
+	COMMAND ${LUA_BINARY} generate_bindings.lua
+	BYPRODUCTS ${CMAKE_SOURCE_DIR}/src/lua/catabindings.cpp
```

Comment:
BYPRODUCTS argument in ADD_CUSTOM_TARGET command requires CMake 3.2 but required version of CMake is set to 2.8